beacon_node/http_api/src/proposer_duties.rs

```diff
@@ -208,7 +227,10 @@ fn compute_historic_proposer_duties<T: BeaconChainTypes>(
         .map_err(BeaconChainError::from)
         .map_err(warp_utils::reject::beacon_chain_error)?;
 
-    convert_to_api_response(chain, epoch, dependent_root, indices)
+    let execution_optimistic = StateId::from_slot(epoch.start_slot(T::EthSpec::slots_per_epoch()))
```

Comment:
As with the attester duties, I think we can just use the exec status of the head block here too.